python/ray/scripts/scripts.py

```diff
@@ -1406,6 +1413,7 @@ def rsync_up(cluster_config_file, source, target, cluster_name, all_nodes):
 )
 @click.argument("script_args", nargs=-1)
 @add_click_logging_options
+@PublicAPI
 def submit(
```

Comment:
Hmm this might be deprecated by jobs. @edoakes @simon-mo should we still recommend submit?

Reply:
I don't think we should be recommending it in the post-jobs era, but jobs is not yet GA (until probably 1.13) and I'm sure we have a lot of people using this command

maybe we can leave it not documented as public api given that we intend to deprecate?